src/Sdk/routing.ts

```diff
@@ -0,0 +1,3 @@
+export const BASE_URL = 'https://api.prezly.com';
+
+export const coverageUrl = '/v2/coverage';
```

Comment:
Seems inconsistent
```suggestion
export const COVERAGE_URL = '/v2/coverage';
```

Reply:
I did this deliberatley to match our `appRouting`. 

The decision to make `BASE_URL` constant case is because I see it as a different type. I can move it to `Sdk.ts` because it's only used there as the default value.

Reply:
Changed 0c85b29578e2fa7434fbff595688c14111942abb